client/src/hooks/useAddChat.ts

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+
+interface User {
+  id: string;
```

Comment:
To trzeba poprawić, bo z bazki leci mongoID
w `types` jest specjalny typ pod to i się nazywa `ID`.

Możesz go tu użyć, bo w reszcie appki mamy ten usage i w innych komponentach się będzie to psuło.